storagenode/storagenodedb/satellites.go

```diff
@@ -17,3 +24,41 @@ const SatellitesDBName = "satellites"
 type satellitesDB struct {
 	migratableDB
 }
+
+// initiate graceful exit
```

Comment:
These comments should probably begin with the name of the function (I think the linter will complain otherwise)